doc/testing.rst

```diff
@@ -0,0 +1,79 @@
+Automating the tests for your App
+=================================
+
+This an introductory guide to writing automated tests for your
+Morepath app.  We assume you've already installed Morepath; if not,
+see the :doc:`installation` section.
+
+In order to carry out the test we'll use WebTest_, which you'll need
+to have installed:
+
+.. code-block:: sh
+
+  $ pip install webtest
+
+
+Testing a basic app
+-------------------
+
+Let’s look at a minimal test of a basic app in Morepath:
+
+.. testcode::
+
+  def test_basic():
+      from morepath.tests.fixtures import basic
+      from webtest import TestApp as Client
+
+      c = Client(basic.app())
```

Comment:
Depending on Morepath fixtures in the test isn't a good idea, I think. Instead it's better to just write a sample app in this text. (besides, the basic fixture is out of date and uses a lowercase app)